### DISCUSSION TITLE
How to stop click propagation on q-select

### DISCUSSION THREAD

**Discussion by LadIQe** [quasarframework/quasar#10248] [General - Components / Directives / etc] [answered]
Hi people,

I have q-table. Some columns have custom header, where is input for filter, like this

```
<template #header-cell-title="props">
  <q-th :props="props">
    <q-input
      dense
      :label="props.col.label"
      hide-bottom-space
      class="table-input"
      @click.stop=""
      v-model="filters.filter"
    />
  </q-th>
</template>
```

When I click on this input to edit it, I had problem that this triggered sort function, so thats why I added `@click.stop=""` and now it works as it should. But I have also q-select, like this

```
<template #header-cell-type="props">
  <q-th
    :props="props"
    @click.stop=""
  >
    <q-select
      dense
      :label="props.col.label"
      hide-bottom-space
      class="table-input"
      v-model="filters.pageType"
      @click.stop=""
      @mousedown.stop=""
      @mouseup.stop=""
      :options="pageTypes"
      style="width: 140px"
      option-value="code"
      option-label="label"
      map-options
      emit-value
      clearable
    />
  </q-th>
</template>
```

And as you can see, I have problem to stop propagation. So it always trigger sort function when I want to click on q-select and see available options in popup.

Do you know how to stop this?

Thanks

**Comment by ontwikkelfabriek** [comment]
@LadIQe 

Please create a codepen.io that demos your issue for better support.

**Reply by LadIQe** [reply]
Hello,

here is sample [code](https://codepen.io/ladiqe/pen/NWjMvzP)

**Comment by pdanpdan** [comment, marked as answer]
you must wrap it in a div and stop the event at that level
https://codepen.io/pdanpdan/pen/xxdjpNj?editors=1010

**Reply by LadIQe** [reply]
Ah I should think of that 😀easy fix, many thanks
